src/XRP/raw-transaction-status.ts

```diff
@@ -30,7 +30,10 @@ export default class RawTransactionStatus {
     return new RawTransactionStatus(
       getTransactionResponse.getValidated(),
       getTransactionResponse.getMeta()?.getTransactionResult()?.getResult(),
-      getTransactionResponse.getTransaction()?.getLastLedgerSequence(),
+      getTransactionResponse
+        .getTransaction()
+        ?.getLastLedgerSequence()
+        ?.getValue(),
```

Comment:
I _think_ this is the right thing to do here?

Reply:
Yup, it is.